x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/shards/LocalStateAutoscalingAndSearchableSnapshots.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.autoscaling.shards;
+
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.license.XPackLicenseState;
+import org.elasticsearch.xpack.autoscaling.LocalStateAutoscaling;
+import org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshots;
+
+public class LocalStateAutoscalingAndSearchableSnapshots extends LocalStateAutoscaling {
```

Comment:
For my own edification and/or someone else reading this code, can you add a comment about why this plugin wrapper is required?

Reply:
I added a comment in 1c4c4aee16fd36ad5f60eb117efe323b0ec10cf6

No expert on this and any conventions. I think adding a `LocalStateSearchablesnapshots` here and using both `LocaState` plugins from the test could also work, but I found this nicer.